servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -63,19 +63,19 @@
 import tachyon.master.journal.JournalEntry;
 import tachyon.master.journal.JournalOutputStream;
 import tachyon.thrift.BlockInfo;
-import tachyon.thrift.BlockInfoException;
+import tachyon.exception.BlockInfoException;
```

Comment:
makes sure imports are sorted alphabetically ... not sure what IDE do you use, but IntelliJ has a function can do this for a given source tree